src/methods.jl

```diff
@@ -1,5 +1,36 @@
 # helper function
 axes_min_length(idxs) = argmin([a isa Colon ? Inf : length(a) for a in idxs])
+axes_min_length(idxs::String) = axes_min_length(fits_indices(idxs))
```

Comment:
why do you need this?

Reply:
For the calculation of `dims` in the case of overscan subtraction.

Reply:
you shouldn't need this. By the time you call the real `subtract_overscan!` function the indices should already be transformed.

Reply:
Interesting, I didn't notice this!